spacy_llm/tasks/ner.py

```diff
@@ -92,12 +90,91 @@ class NERTask:
 Text:
 '''
 {{ text }}
+'''
+"""
+
+
+@registry.llm_tasks("spacy.NER.v1")
+def make_ner_task_v1(
+    labels: str,
+    examples: Optional[Callable[[], Iterable[Any]]] = None,
+    normalizer: Optional[Callable[[str], str]] = None,
+    alignment_mode: Literal["strict", "contract", "expand"] = "contract",  # noqa: F821
+    case_sensitive_matching: bool = False,
+    single_match: bool = False,
+):
+    task = NERTask(
+        labels=labels,
+        examples=examples,
+        normalizer=normalizer,
+        alignment_mode=alignment_mode,
+        case_sensitive_matching=case_sensitive_matching,
+        single_match=single_match,
+    )
+    setattr(task, "_TEMPLATE_STR", _NER_TEMPLATE_V1)
+    return task
+
+
+@registry.llm_tasks("spacy.NER.v2")
```

Comment:
Let's go ahead and move this into a separate method, too? I think it will be easier to maintain if you don't have constrain the `__init__` to correspond to the registered function.

Reply:
Agreed, but see #81

Reply:
Okay, that can wait then.